src/controllers/search.controller.js

```diff
@@ -0,0 +1,18 @@
+const searchService = require('../services/search.service');
+
+const searcH = async(req,res) => {
+    try{
+        const name = req.params.categories_name;
```

Comment:
```suggestion
        const { categoryId } = req.params;
```
dao layer에서 작성된 sql문을 보면 where clause에 결국 사용되는 조건은 `WHERE p.category_id = ?;` 이와 같은데 name이라고 이름짓고 받는 것은 맥락이 맞지 않는 것 같습니다. 변수명도 수정해주시고 req.params로 받는 값도 수정해주세요~!